draft-ietf-httpbis-bcp56bis.md

```diff
@@ -541,7 +541,7 @@ When used, it is important to carefully specify the scoping and use of cookies;
 
 Clients often need to send multiple requests to perform a task.
 
-In HTTP/1 {{HTTP11}}, parallel requests are most often supported by opening multiple connections. Application performance can be impacted when too many simultaneous connections are used, because connections' congestion control will not be coordinated. Furthermore, it can be difficult for applications to decide when and to select which connection to use to issue a given request, further impacting performance.
+In HTTP/1 {{HTTP11}}, parallel requests are most often supported by opening multiple connections. Application performance can be impacted when too many simultaneous connections are used, because connections' congestion control will not be coordinated. Furthermore, it can be difficult for applications to decide when to issue and to select which connection to use to issue a given request, further impacting performance.
```

Comment:
```suggestion
In HTTP/1 {{HTTP11}}, parallel requests are most often supported by opening multiple connections. Application performance can be impacted when too many simultaneous connections are used, because connections' congestion control will not be coordinated. Furthermore, it can be difficult for applications to decide when to issue and which connection to use for a given request, further impacting performance.
```

Reply:
A more elegant solution, thanks.